diff --git a/node_modules/@nativescript/core/platforms/android/core.aar b/node_modules/@nativescript/core/platforms/android/core.aar
new file mode 100644
index 0000000..09b3af4
Binary files /dev/null and b/node_modules/@nativescript/core/platforms/android/core.aar differ
diff --git a/node_modules/@nativescript/core/ui/image/image-common.js b/node_modules/@nativescript/core/ui/image/image-common.js
index 4d1713b..0b53c98 100644
--- a/node_modules/@nativescript/core/ui/image/image-common.js
+++ b/node_modules/@nativescript/core/ui/image/image-common.js
@@ -154,13 +154,13 @@ export const stretchProperty = new Property({
     affectsLayout: __APPLE__,
 });
 stretchProperty.register(ImageBase);
-export const tintColorProperty = new InheritedCssProperty({
+export const tintColorProperty = new Property({
     name: 'tintColor',
     cssName: 'tint-color',
     equalityComparer: Color.equals,
     valueConverter: (value) => new Color(value),
 });
-tintColorProperty.register(Style);
+tintColorProperty.register(ImageBase);
 export const decodeHeightProperty = new Property({
     name: 'decodeHeight',
     defaultValue: { value: 0, unit: 'dip' },
